README.md

```diff
@@ -0,0 +1,173 @@
+      {
+        "source": {
+          "func": "url_read",
+          "params": {
+            "path": "https://opendata.vancouver.ca/api/explore/v2.1/catalog/datasets/public-art/exports/json?lang=en&timezone=America%2FLos_Angeles"
+          }
+        },
+        "filters": [
+          {
+            "func": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/filters/canada/british-columbia/vancouver/public-art-json-to-json.js",
+            "params": {
+              "library": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/libraries/standard.js"
+            }
+          },
+          {
+            "func": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/filters/stringify.js",
+            "params": {
+              "indent": true
+            }
+          }
+        ],
+        "sinks": [
+          {
+            "func": "file_write",
+            "params": {
+              "path": "./my-vancouver-data-1.json"
+            }
+          },
+          {
+            "func": "file_write",
+            "params": {
+              "path": "./my-vancouver-data-2.json"
+            }
+          }
+        ]
+      },
+      {
+        "source": {
+          "func": "url_read",
+          "params": {
+            "path": "https://data.austintexas.gov/resource/uuk6-933w.json"
+          }
+        },
+        "filters": [
+          {
+            "func": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/filters/united-states/texas/austin/public-art-json-to-json.js",
+            "params": {
+              "library": "https://raw.githubusercontent.com/OpendataDeveloperNetwork/ODEN-Transmogrifiers/dev/libraries/standard.js"
+            }
+          }
+        ]
+      }
+    ]
+  }
+]
+```
+
+### How to Run a Manifest with the Transmogrifier
+
+To run the transmogrifier, you need an executor file, such as this [example](https://github.com/transmogrifier-io/transmogrifier/blob/dev/example-main.js). 
+The example transmogrifier takes the manifest as a command line argument, so to initiate execution with node, for example, you will run the following command:
+(assuming you are in the same directory as the example-main.js) 
+`node example-main.js my_manifest.json` 
+
+# Implementations
+This repository includes implementations for the transmogrifier in both Javascript and Dart.
+
+# Limitations
+The following sections cover any limitations that the different transmogrifier implemenations may have.
+
+## Javascript implementation limitations
+The Javascript implementation uses node.js for some of its functionality, applications that are developed using flutter will
+not be able to use this implementation. The Dart implementation should be used if this is the case.
+
+## Dart implementation limitations
+The Dart implementation currently does not support source/sink functions from a URL, only from a local path.
```

Comment:
think it might be better to word as "The Dart implementation currently only supports builtin source/sink functions"